oozie-to-airflow/mappers/file_mixin.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict
+
+
+class FileMixin:
+    def __init__(self, params: Dict[str, str]):
+        self.files = None
+        self.hdfs_files = None
+        self.params = params
+
+    def _preprocess_path_to_hdfs(self, path: str):
+        if path.startswith("/"):
+            return self.params["nameNode"] + path
+        else:
+            return self.params["oozie.wf.application.path"] + "/" + path
+
+    @staticmethod
+    def _check_path_for_comma(path: str) -> None:
+        """
+        Raises exception if path has comma
+        :param path: path to check
+        :return: None
+        """
+        if "," in path:
+            raise Exception("There should not be ',' in the path {}".format(path))
+
+    @staticmethod
+    def _check_path_for_hash(path: str) -> [str]:
```

Comment:
Maybe we could rename it to `_check_path_for_hash_sign()`?
This way it's clear that we're merely looking for the hash sign in a string, and some hash value.

Reply:
Now, looking at what the method does exactly, I'd even lean towards a name like `_split_by_hash_sign()` -> that also clearly states that we're splitting the string.

Reply:
Agree. Done